.github/CODEOWNERS

```diff
@@ -16,12 +16,22 @@
 /_maps/ @Jolly-66
 
 #Tajaran
-talestation_modules/code/modules/mob/living/carbon/human/species_types/tajaran.dm @Jolly-66
+/talestation_modules/code/modules/mob/living/carbon/human/species_types/tajaran.dm @Jolly-66
 
 #prodirus
 #Pain
-talestation_modules/code/__DEFINES/pain_defines.dm @prodirus
-talestation_modules/code/datums/pain/pain_helpers.dm @prodirus
-talestation_modules/code/datums/pain/pain.dm @prodirus
-talestation_modules/code/modules/mob/living/carbon/pain/ @prodirus
-talestation_modules/code/modules/reagents/chemistry/reagents/pain_reagents.dm @prodirus
+/talestation_modules/code/__DEFINES/pain_defines.dm @prodirus
+/talestation_modules/code/datums/pain/pain_helpers.dm @prodirus
+/talestation_modules/code/datums/pain/pain.dm @prodirus
+/talestation_modules/code/modules/mob/living/carbon/pain/ @prodirus
+/talestation_modules/code/modules/reagents/chemistry/reagents/pain_reagents.dm @prodirus
```

Comment:
Could you atomize these out? You can, in theory, still leave it in the PR, but I'm asking if you can make a separate PR for just the codeowners tidying.

Reply:
It's just adding a / at the stat so they work